### PR TITLE
Task/juan n14

### PR DESCRIPTION
**Pull request by juanplusjuan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "react-redux": "^5.1.1",
     "react-router": "^4.4.0-beta.6",
     "react-router-dom": "^4.4.0-beta.6",
-    "react-scroll": "^1.7.11",
+    "react-router-hash-link": "^1.2.1",
```
Comment (lkazberova) [inline package.json]: @juanplusjuan can you please let me know why you decide to change package? Why I ask, we also have an opportunity to use hash in url with react-scroll. But react-scroll also provide ability to set offset for scrolling and we need it. If you take a look at result of scrolling to About us you will see that caption went up and not visible.

Comment (lkazberova) [reply package.json]: Also in react-scroll we can style "selected" state of Link and we used it

File: src/screens/Site/PrivacyPolicy.jsx [hunk 2]
```diff
@@ -1,245 +1,288 @@
...
-        <Typography variant="body1">
-            By using our Service you consent to our collection and use of your
-            information as described in this Privacy Policy. If we change our
-            privacy policies and procedures, we will post those changes on
-            http://sayprimer.com and other services to keep you aware of what
-            information we collect, how we use it and under what circumstances
-            we may disclose it.
-        </Typography>
-    </main>
+    <HeaderFooterLayout>
```
Comment (lkazberova) [inline src/screens/Site/PrivacyPolicy.jsx]: I suppose we should think about background for header in this case. Before scrolling we can't see header's items

Comment (juanplusjuan) [reply src/screens/Site/PrivacyPolicy.jsx]: @lkazberova good spot.  I fixed it.

File: package.json (hunk 1 above)
Comment (juanplusjuan) [reply package.json]: @lkazberova I played with `react-scroll` for a bit and I couldn't get it to work across pages.  Try clicking on "Products" in the menu while you're on Terms of Service and you'll see it navigates to the home page and then scrolls down to Products.

**Pull request merged**